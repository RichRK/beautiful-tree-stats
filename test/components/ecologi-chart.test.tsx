import React from 'react'
import { render } from '@testing-library/react'
import EcologiChart from '../../components/ecologi-chart'
import TreeDate from '../../models/tree-date-model'
import { DateTime } from 'luxon'

const treeData: TreeDate[] = [
  { date: DateTime.now(), quantity: 5 },
  { date: DateTime.now(), quantity: 50 },
  { date: DateTime.now(), quantity: 500 },
]

describe('EcologiChart component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <EcologiChart
        chartType="line"
        data={treeData}
        primaryDataKey="quantity"
        labelKey="date"
      />,
      {}
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
