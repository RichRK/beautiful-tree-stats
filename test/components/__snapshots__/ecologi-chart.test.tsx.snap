// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EcologiChart component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="recharts-responsive-container"
    style="width: 100%; height: 300px;"
  />
</DocumentFragment>
`;
